_data/simple-icons.json

```
@@ -3675,6 +3675,11 @@
             "hex": "3F5767",
             "source": "https://coveralls.io"
         },
+        {
+            "title": "Coze",
+            "hex": "4F54DF",
```

```suggestion
            "hex": "4D53E8",
```
Color from logo on their website header